### PR TITLE
remove display interface

### PR DESCRIPTION
**Pull request by kimhyungrok**
As deciding not to include display interface in nugu_sdk,
it remove related interface and agent, listener.

Signed-off-by: Hyungrok.Kim <hr97gdi@sk.com>

### CODE REVIEW

File: include/capability/audio_player_interface.hh [hunk 1]
```diff
@@ -56,11 +55,112 @@ enum class RepeatType {
...
+    virtual void stopRenderingTimer(const std::string& id) = 0;
+};
+
 /**
  * @brief audioplayer listener interface
  * @see IAudioPlayerHandler
  */
-class IAudioPlayerListener : virtual public IDisplayListener {
+class IAudioPlayerListener : public ICapabilityListener,
+                             public IDisplayRenderingListener {
```
Comment (JeanTracker) [inline include/capability/audio_player_interface.hh]: We decided to handle the display later. Please remove inheritance of `class IDisplayRenderingListener` as below
```
class IAudioPlayerListener : public ICapabilityListener
```

File: include/capability/audio_player_interface.hh [hunk 2]
```diff
@@ -120,7 +220,8 @@ public:
  * @brief audioplayer handler interface
  * @see IAudioPlayerListener
  */
-class IAudioPlayerHandler : virtual public IDisplayHandler {
+class IAudioPlayerHandler : virtual public ICapabilityInterface,
+                            public IDisplayRenderingHandler {
```
Comment (JeanTracker) [inline include/capability/audio_player_interface.hh]: Please remove inheritance of `class IDisplayRenderingHandler` same reason.

File: include/capability/audio_player_interface.hh (hunk 1 above)
Comment (kimhyungrok) [reply include/capability/audio_player_interface.hh]: I agree. I apply it.

File: include/capability/audio_player_interface.hh (hunk 2 above)
Comment (kimhyungrok) [reply include/capability/audio_player_interface.hh]: I agree. I apply it.

**Pull request merged**